### PR TITLE
Remove Lux from sources

### PR DESCRIPTION
**Pull request by bidzyyys**

### CODE REVIEW

**Review by nieznanysprawiciel — APPROVED** [review]

File: tests/golem/docker/test_docker_manager.py [hunk 1]
```diff
@@ -293,11 +293,11 @@ def command(key, *args, **kwargs):
 
         from apps.core import nvgpu
         if nvgpu.is_supported():
-            expected = 6
+            expected = 5                #before removing lux 6
```
Comment (Krigpl) [inline tests/golem/docker/test_docker_manager.py]: These comments aren't useful imo

**Review by Krigpl — COMMENTED** [review]
itshappening

File: tests/golem/docker/test_docker_manager.py (hunk 1 above)
Comment (magdasta) [reply tests/golem/docker/test_docker_manager.py]: +1

Comment (bidzyyys) [reply tests/golem/docker/test_docker_manager.py]: Comments removed

**Review by Elfoniok — CHANGES_REQUESTED** [review]
Please check problem with building binaries on Windows.

**Review by Elfoniok — APPROVED** [review]

**Review by Krigpl — APPROVED** [review]
mergeitmergeitmergeit

**Pull request merged**